.travis.yml

```diff
@@ -5,6 +5,8 @@ node_js:
 
 cache:
   ccache: true
+  directories:
+    - node_modules
```

Comment:
Docker mirrors every file that it creates on the current directory, so `node_modules` (as well as `dist`) end up in the right place regardless.

> Hopefully re-enabling this won't uncover the same bug that @jhermsmeier hit ;-)

Haha, lets hope for the best :D